screens/home/components/members-list.tsx

```diff
@@ -1,45 +1,59 @@
-        <MembersListItem title='Aras Atasaygin' designation='Engineering Manager' company='Altayer' imageUrl='/avatars/aras.jpg' />
-        <MembersListItem title='Pavlo Lompas' designation='Senior Software Engineer' company='Tradeling' imageUrl='/avatars/pavlo.jpeg' />
-        <MembersListItem title='Umar Farooq' designation='Lead Engineer' company='Mart' imageUrl='/avatars/umarmehar.jpeg' />
-        {/*<MembersListItem title='Mohamed Meabed' designation='Chief Technology Officer' company='Tradeling' imageUrl='/avatars/meabed.png' />*/}
-        {/*<MembersListItem title='Ahmed Moawad' designation='VP of Engineering' company='Tradeling' imageUrl='/avatars/ahmed.png' />*/}
-        <MembersListItem title='Ming Hu' designation='Software Engineer' company='Tradeling' imageUrl='/avatars/ming.jpeg' />
-        <MembersListItem title='Segun Adebayo' designation='Design System Engineer' company='Tradeling' imageUrl='/avatars/segun.jpeg' />
-        <MembersListItem title='Umar Mehmood' designation='Frontend Engineer' company='AlTayer' imageUrl='/avatars/umar.jpg' />
-        <MembersListItem title='Furqan Zafar' designation='FullStack Engineer' company='AlTayer' imageUrl='/avatars/furqan.jpeg' />
-        <MembersListItem title='Martian Lee' designation='Lead Frontend Engineer' company='Tradeling' imageUrl='/avatars/martian.jpeg' />
-        <MembersListItem title='Numan Mehmood' designation='Product Designer' company='Tradeling' imageUrl='/avatars/noman.jpeg' />
-        <MembersListItem title='Mustafa Magdi' designation='Engineering Manager' company='Tradeling' imageUrl='/avatars/mustafa.jpeg' />
-        <MembersListItem title='Hany' designation='Software Engineer' company='Tradeling' imageUrl='/avatars/hany.jpeg' />
-        <MembersListItem title='Tauqeer Nasir' designation='Software Engineer' company='Tradeling' imageUrl='/avatars/tauqeer.jpeg' />
-        <MembersListItem title='Amad' designation='Software Engineer' company='Tradeling' imageUrl='/avatars/amad.jpeg' />
-        <MembersListItem title='Abdallah' designation='Software Engineer' company='Tradeling' imageUrl='/avatars/abdallah.jpeg' />
-        <MembersListItem title='Mohamed Sayed' designation='Software Engineer' company='Tradeling' imageUrl='/avatars/maradona.jpeg' />
-        <MembersListItem title='Pranjal Baweja' designation='Software Engineer' company='Tradeling' imageUrl='/avatars/pranjal.jpeg' />
-        { !isExpanded && <Box opacity={ 1 } pointerEvents='none' position='absolute' bottom={ 0 } height='200px' width='100%' background='linear-gradient(180deg, transparent, black)' /> }
+      <SimpleGrid
+        columns={[1, 2, 3, 4]}
+        mt="40px"
+        gridGap="15px"
+        position="relative"
+        maxHeight={isExpanded ? undefined : "750px"}
+        overflow="hidden"
+        className="hover-grayscale"
+      >
+        {members.map((member, i) => (
+          <MembersListItem {...member} key={i} />
```

Comment:
Why not add a memberId property to members.json & use it here for key attribute
We can latter create markdown files with the same memberId property to store detailed member profiles

using index as a key is an anti pattern: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Yes, will do that. @kamranahmedse is working on some other things and refactoring, just waiting for him to finish first.